docs/spec/spec.md

```diff
@@ -50,6 +50,21 @@ Schema: [`Response.schema.json`](../schemas/Response.schema.json)
         ]
     }
 
+# Standard response codes
```

Comment:
Might be nice to just note what each of these mean next to each number? (I couldn't remember what 409 was!) Go ahead and merge if you don't think it's worth it though!